test-suite-redis/src/test/groovy/io/micronaut/crac/CacheSpec.groovy

```diff
@@ -0,0 +1,66 @@
+package io.micronaut.crac
+
+import io.micronaut.cache.annotation.Cacheable
+import io.micronaut.context.annotation.Property
+import io.micronaut.context.annotation.Requires
+import io.micronaut.core.util.StringUtils
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.annotation.Client
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import spock.lang.PendingFeature
+
+@Property(name = "spec.name", value = "CacheSpec")
+@Property(name = "redis.caches.test.enabled", value = StringUtils.TRUE)
+class CacheSpec extends BaseCacheSpecification {
+
+    @Inject
+    @Client("/")
+    HttpClient client
+
+    @PendingFeature(reason = "Currently this doesn't work, as the CacheManager keeps hold of it's cache connections")
+    def "cache annotations work"() {
```

Comment:
does the test verify that cached values are still there upon restoration?

Reply:
Yes, on line 41 but it's a `@PendingFeature` so it fails (as the cache no longer works due to the Redis connection being destroyed)